src/chi/themes/connect/_variables.scss

```
@@ -1151,13 +1151,20 @@ $css-grid-gap: 1rem;
 $css-grid-md-gap: 1.5rem;
 
 // Date picker
-$date-picker-input-width: 15rem;
-$date-picker-day-border-radius: 0.1875rem;
-$date-picker-padding: 0.5rem;
+$date-picker-disabled-icon-color: $color-icon-muted;
+$date-picker-icon-color: $color-icon-muted-dark;
+$date-picker-input-width: 12.75rem;
+$date-picker-margin: 0.5rem;
+$date-picker-padding: 0;
 $date-picker-day-active-background-color: $color-teal-70;
 $date-picker-day-active-text-color: $color-text-white;
-$date-picker-day-background-color: $color-background-white;
+$date-picker-day-border-radius: 0.1875rem;
+$date-picker-day-line-height: 1.5rem;
 $date-picker-day-today-accent-background-color: $color-teal-70;
+$date-picker-day-today-accent-background-color-active: $color-background-white;
+$date-picker-day-today-bottom: 0;
```

In `rem` and dame for test theme.

```suggestion
$date-picker-day-today-bottom: 3px;
```